packages/messaging/src/controllers/controller-interface.ts

```diff
@@ -371,6 +371,7 @@ function isTokenStillValid(
   tokenDetails: TokenDetails
 ): boolean {
   if (
+    !tokenDetails.vapidKey ||
```

Comment:
do we always save the VAPID key? Even if it's the default one?

Reply:
I updated the PR description to explain the issue.